utils/dockerutil.py

```diff
@@ -95,3 +102,39 @@ def find_cgroup_filename_pattern(mountpoints, container_id):
             return os.path.join('%(mountpoint)s/system/docker/%(id)s/%(file)s')
 
     raise MountException("Cannot find Docker cgroup directory. Be sure your system is supported.")
+
+
+def image_tag_extractor(co, key):
+    if "Image" in co:
+        split = co["Image"].split(":")
+        if len(split) <= key:
+            return None
+        elif len(split) > 2:
+            # if the repo is in the image name and has the form 'docker.clearbit:5000'
+            # the split will be like [repo_url, repo_port/image_name, image_tag]. Let's avoid that
+            split = [':'.join(split[:-1]), split[-1]]
+        return [split[key]]
+    if "RepoTags" in co:
```

Comment:
no, `Image` is present for a container, `RepoTag` for an image. But you're right, that's not clear. I'll replace `co` with `entity` to avoid this confusion.